packages/extism/extism.1.3.0/opam

```diff
@@ -0,0 +1,50 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Extism bindings"
+description: "Bindings to Extism, the universal plugin system"
+maintainer: ["Extism Authors <oss@extism.org>"]
+authors: ["Extism Authors <oss@extism.org>"]
+license: "BSD-3-Clause"
+tags: ["topics" "wasm" "plugin"]
+homepage: "https://github.com/extism/ocaml-sdk"
+doc: "https://github.com/extism/ocaml-sdk"
+bug-reports: "https://github.com/extism/ocaml-sdk/issues"
+depends: [
+  "ocaml" {>= "4.14.1"}
+  "dune" {>= "3.2"}
+  "ctypes" {>= "0.22.0"}
+  "ctypes-foreign" {>= "0.22.0"}
+  "integers" {>= "0.3.0"}
+  "bigstringaf" {>= "0.9.0"}
+  "ppx_yojson_conv" {>= "v0.15.0"}
+  "extism-manifest" {= version}
+  "ppx_inline_test" {>= "v0.15.0"}
+  "cmdliner" {>= "1.1.1"}
+  "uuidm" {>= "0.9.8"}
+  "mdx" {>= "2.3.0" & with-test}
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/extism/ocaml-sdk.git"
+build-env: [EXTISM_TEST_NO_LIB = "1"]
```

Comment:
I think build-env is only available in opam 2.1.5 and higher, but this is probably good enough as it only affects tests.